src/args.jl

```diff
@@ -1283,6 +1285,8 @@ const _match_map2 = KW(
     :fontfamily_subplot       => :fontfamily,
     :tickfontfamily           => :fontfamily_subplot,
     :guidefontfamily          => :fontfamily_subplot,
+    :foreground_color_tick                => :foreground_color_subplot,
+    :foreground_color_minortick           => :foreground_color_subplot
```

Comment:
I would add these to `_match_map` instead and let them default to `:foreground_color_axis` (which then defaults to `:foreground_color_subplot`). Then you should be able to have different tick colors for different axis.

Reply:
hey @greimel would you mind doing this minor change so we can get this merged? Thanks!